gramps/gen/plug/docgen/treedoc.py

```diff
@@ -815,7 +816,14 @@ def close(self):
             else:
                 proc = Popen(args, stdin=PIPE, stdout=PIPE, stderr=PIPE)
```

Comment:
The current change isn't enough to fully fix [10696](https://gramps-project.org/bugs/view.php?id=10696). By changing the call to Popen() in Windows to match the other platforms, report generation works on Windows.

Reply:
Thanks.  I have changed this, but cannot test it because I don't have a copy of Windows.

Reply:
@Nick-Hall I'll be able to test the change on Windows 10 and 11 in a couple of days and report results.

Reply:
@Nick-Hall Tested on both Win 11 and 10 and unfortunately report generation continues to fail. When this issue was being discussed on the forums, the local change I had did seem to resolve it. Will need experiment some more.

It would be helpful if anyone else could test on Windows to confirm.